automation/jenkins_generate_ami.sh

```diff
@@ -0,0 +1,52 @@
+#/bin/bash
+
+if [ "$deployTo" = "production" ]; then
+    S3_ACCESS_KEY=${PRODUCTION_S3_ACCESS_KEY}
+    S3_SECRET_KEY=${PRODUCTION_S3_SECRET_KEY}
+    S3_REGION=${PRODUCTION_S3_REGION:-us-east-1}
+    BALENA_PRELOAD_SSH_PUBKEY=${PRELOAD_SSH_PUBKEY_PRODUCTION}
+    BALENACLI_TOKEN=${BALENAOS_PRODUCTION_TOKEN}
+    BALENA_ENV="balena-cloud.com"
+elif [ "$deployTo" = "staging" ]; then
+    S3_ACCESS_KEY=${STAGING_S3_ACCESS_KEY}
+    S3_SECRET_KEY=${STAGING_S3_SECRET_KEY}
+    S3_REGION=${STAGING_S3_REGION:-us-east-1}
+    BALENA_PRELOAD_SSH_PUBKEY=${PRELOAD_SSH_PUBKEY_STAGING}
+    BALENACLI_TOKEN=${BALENAOS_STAGING_TOKEN}
+    BALENA_ENV="balena-staging.com"
+fi
+
+MACHINE=${JOB_NAME#yocto-}
+YOCTO_IMAGES_PATH="$WORKSPACE/build/tmp/deploy/images/${MACHINE}"
+
+# TODO: Replace the default value with the value read from the CoffeeScript file once available
+IMAGE_NAME=${IMAGE_NAME:-resin-image-genericx86-64-ext.resinos-img}
```

Comment:
If the jenkins job clones the balena-intel repo, the .coffee file is already available for you to extract this, isn't it?

Reply:
Currently, coffeescript does not contain the reference to the raw image yet (there's a PR in progress). However, just got to know today that we also need a higher version of the supervisor in the image, so probably everything will be in place with the new version and we won't need this default